utilities/EluvioCli.js

```diff
@@ -17,23 +14,19 @@ const { ElvFabric } = require("../src/ElvFabric");
 const Utils = require("@eluvio/elv-client-js/src/Utils");
 const { ElvUtils } = require("../src/Utils");
 const sql = postgres({
-  host: "rep1.elv",
+  host : "rep1.elv",
   port: 5432,
   database: "indexer",
   username: "",
-  password: "",
+  password: ""
```

Comment:
i think we prefer trailing commas???   seems there are a lot of changes to remove them, confused about that.  i think we need to fix some linters or whatever is causing this.

if we're gonna do a big change en-mass, i'd say _add_ them instead of removing. or leave as-is for git blame.